### PR TITLE
Convert changelog to markdown file, record 0.6.0 changes (main)

### PR DESCRIPTION
**Pull request by korydraughn**
Merge once @alanking and @d-w-moore report PRC and Ingest tests _(w/ PRC tip-of-main)_ pass.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -409,9 +438,9 @@ v0.6.0 (2017-05-23)
 - Unit test. [Antoine de Torcy]
 - First pass at data object replication. [Antoine de Torcy]
 
+## [v0.5.0] - 2016-08-15
+
```
Comment (trel) [inline CHANGELOG.md:442]: remove extra newline

**Comment by trel** [comment]
please also look at setup.cfg... i think it should point to README.md

**Comment by trel** [comment]
and look at MANIFEST.in

**Comment by trel** [comment]
and AUTHORS

**Comment by trel** [comment]
and consider removing the auto-generator... `.gitchangelog.rc`

File: CHANGELOG.md (hunk 1 above)
Comment (korydraughn) [reply CHANGELOG.md:442]: Done

**Comment by korydraughn** [comment]
What do you want to do about the AUTHORS file? Doesn't add any value IMO. People can always find it in the git history.

**Comment by korydraughn** [comment]
I think the AUTHORS file can be removed.

**Comment by trel** [comment]
> I think the AUTHORS file can be removed.

Agreed.

**Comment by trel** [comment]
i think all these commits are 'packaging' and should be issue 3 ?

**Comment by trel** [comment]
removing `.gitchangelog.rc` could be 5... or also 3.

**Comment by korydraughn** [comment]
Okay. Will make all of them 3.

**Comment by korydraughn** [comment]
Squashed and reordered.

Did not include recent changes (e.g. AUTHORS file removal) in changelog because the changelog is supposed to highlight notable changes.

**Review by trel — APPROVED** [review]
I think that's fine - it's just packaging, rather than code.

Looks good - let's await another set of eyes.

**Review by alanking — COMMENTED** [review]
Looks good to me. The last commit message says "0.6.0 release" when I think it should say "2.2.0 release", right?

**Comment by korydraughn** [comment]
Fixed it!

**Review by alanking — APPROVED** [review]
\# em

**Comment by d-w-moore** [comment]
Tests pass on tip of main, with the exception of `test_assigning_session_connection_timeout_to_invalid_values__issue_569` which just needs float('Inf') removed as a test value

**Comment by alanking** [comment]
@d-w-moore - https://github.com/irods/python-irodsclient/pull/638 was merged earlier today, so that issue should be resolved now.

I will also confirm that ingest tests were passing when running with a pip package based on commit https://github.com/irods/python-irodsclient/commit/dcd6c135c81b8fba16f3fda4b7e7fcc1f5b012bf. The only difference between tip of main and that commit is the aforementioned removal of `float(Inf)` from a test, which should not have any effect on ingest.

**Comment by korydraughn** [comment]
#'d

**Pull request merged**